cpp/ql/src/Critical/DoubleFreeGood2.cpp

```diff
@@ -0,0 +1,17 @@
+void g() {
+	MyTask *task = NULL;
+
+	try
+	{
+		task = new MyTask;
+
+		...
+
+		delete task;
+		task = NULL;
```

Comment:
I think we should be suggesting `nullptr` since this is in a `.cpp` file. We could always mention that `NULL` should be used in C.

Reply:
Done.  I'm not too worried about explaining the `NULL` / `nullptr` distinction in the text, that's getting a bit off topic IMO.